module/VuFind/src/VuFind/ILS/Driver/Demo.php

```diff
@@ -1542,6 +1542,28 @@ public function getCourses()
         return $this->courses;
     }
 
+    /**
+     * Get a set of random bib IDs
+     *
+     * @param int $limit Maximum number of IDs to return (max 30)
+     *
+     * @return string[]
+     */
+    protected function getRandomBiBIds($limit): array
```

Comment:
I suppose the capital B in `BiB` is a typo, right?

Reply:
Ha, yes! Case insensitive function names in PHP can be a nuisance sometimes. Fixed now. :-)